aeflex/aeflex.go

```diff
@@ -49,9 +53,19 @@ type AEFlexSource struct {
 }
 
 // NewAEFlexSource returns a new discovery object with an authenticated AppEngine client.
-func NewAEFlexSource(project string) (*AEFlexSource, error) {
+func NewAEFlexSource(project, filename string) *AEFlexSource {
```

Comment:
Maybe call this NewDiscoverySource?  AEFlex is redundant.  I see that means changing the struct name as well.  8-(